package.json

```
@@ -46,6 +47,7 @@
     "jayson": "3.6.5",
     "jsonwebtoken": "9.0.0",
     "merkle-patricia-tree": "4.2.4",
+    "pino": "^9.6.0",
```

please use fixed version
```suggestion
    "pino": "9.6.0",
```